File: src/main/java/de/tectoast/emolga/commands/pokemon/GerCommand.java
package de.tectoast.emolga.commands.pokemon;

import de.tectoast.emolga.commands.Command;
import de.tectoast.emolga.commands.CommandCategory;
import de.tectoast.emolga.commands.GuildCommandEvent;

public class GerCommand extends Command {
    public GerCommand() {
        super("ger", "Zeigt den deutschen Namen dieser Sache.", CommandCategory.Pokemon);
        setArgumentTemplate(ArgumentManagerTemplate.builder()
                .add("stuff", "Pokemon|Attacke|Fähigkeit|Item|Typ", "Die Sache, von der du den englischen Namen haben möchtest", Translation.Type.all())
                .setExample("!ger Primarina")
                .build());
    }

    @Override
    public void process(GuildCommandEvent e) {
        String stuff = e.getArguments().getTranslation("stuff").getTranslation();
        if(stuff.equals("Psychokinese")) {
            e.reply("Psychokinese/Psycho");
            return;
        }
        e.reply(stuff);
    }
}

File: src/main/kotlin/de/tectoast/emolga/commands/pokemon/GerCommand.kt
package de.tectoast.emolga.commands.pokemon

import de.tectoast.emolga.commands.Command
import de.tectoast.emolga.commands.CommandCategory
import de.tectoast.emolga.commands.GuildCommandEvent
import de.tectoast.emolga.commands.Translation

class GerCommand : Command("ger", "Zeigt den deutschen Namen dieser Sache.", CommandCategory.Pokemon) {
    init {
        argumentTemplate = ArgumentManagerTemplate.builder()
            .add(
                "stuff",
                "Pokemon|Attacke|Fähigkeit|Item|Typ",
                "Die Sache, von der du den englischen Namen haben möchtest",
                Translation.Type.all()
            )
            .setExample("!ger Primarina")
            .build()
    }

    override suspend fun process(e: GuildCommandEvent) {
        val stuff = e.arguments.getTranslation("stuff").translation
        e.reply(if (stuff == "Psychokinese") "Psychokinese/Psycho" else stuff)
    }
}
